### PR TITLE
chore(docs): update docs content

### PR DESCRIPTION
**Pull request by dbritto-dev**
## Summary

Update docs content

## Check List

- [x] `pnpm run prettier` for formatting code and docs

### CODE REVIEW

File: docs/guides/how-to-reset-state.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ const useSlice = create<State & Actions>()((set, get) => ({
 Resetting multiple stores at once
 
 ```ts
-import { create as _create } from 'zustand'
-import type { StateCreator } from 'zustand'
+import type * as zustand from 'zustand'
```
Comment (dai-shi) [inline docs/guides/how-to-reset-state.md:47]: Personally, I'm not a big fan of this. Can we keep the previous style?

Comment (dbritto-dev) [reply docs/guides/how-to-reset-state.md:47]: Sure

**Comment by dbritto-dev** [comment]
@dai-shi all yours

File: docs/guides/testing.md [hunk 2]
```diff
@@ -83,8 +83,9 @@ In the next steps we are going to setup our Jest environment in order to mock Zu
 
 ```ts
 // __mocks__/zustand.ts
-import * as zustand from 'zustand'
 import { act } from '@testing-library/react'
+import type * as zustand from 'zustand'
```
Comment (dai-shi) [inline docs/guides/testing.md:87]: Can you do the same for others please?

Comment (dbritto-dev) [reply docs/guides/testing.md:87]: In this case we use it to type the import from `requireActual` or

Comment (dbritto-dev) [reply docs/guides/testing.md:87]: @dai-shi keep me in the loop

Comment (dai-shi) [reply docs/guides/testing.md:87]: Oh, I see L91.

Can we do something like this? Feels a little more explicit.
```suggestion
import type * as ZustandExportedTypes from 'zustand'
```

Comment (dbritto-dev) [reply docs/guides/testing.md:87]: @dai-shi sure

**Comment by dbritto-dev** [comment]
@dai-shi all yours

**Review by dai-shi — APPROVED** [review]
Thanks. I'm not sure if it's common practice, so we can change it if someone has suggestions.

**Comment by loddit** [comment]
I noticed that `xIsNext` might not be necessary as an independent state ([React State Design Principle](https://react.dev/learn/thinking-in-react#step-3-find-the-minimal-but-complete-representation-of-ui-state)). The Board component can derive `player` from squares, for example:
```
const player = (squares.filter(v => ['O', 'X'].includes(v)).length % 2) ? 'O' : 'X'
```
was it designed this way for a specific reason?

**Pull request merged**